feature/interface/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/README.md

```diff
@@ -0,0 +1,51 @@
+# RT-5.x: Interface Loopback mode
+[TODO] assign test number 
+
+## Summary
+
+Validate IPv6 ND Router Advertisement (RA) could be completely disabled.
+
+## Procedure
+*   Connect DUT porr-1 to OTG port-1
+*   Configure IPv6 address on DUT port-1
+*   set IPv6 ND RA transmission interval on DUT port-1 to 5 seconds
+*   disable IPv6 ND RA transmission on DUT port-1
+
+### TestCase-1: No periodical Router Advertisement
+
+*   Observe over period of 10 seconds if IPv6 ND RA arrives on OTG Port-1 ([rfc4861 section 6.2.1](https://datatracker.ietf.org/doc/html/rfc4861#section-6.2.1))
```

Comment:
what is our expectation here? To see RAs or not to see? Let's clarify specifically?